aws_connect.py

```diff
@@ -124,6 +138,17 @@ async def get_current_active_user(
     return current_user
 
 
+def user_in_resource_groups(
+    current_user: Annotated[User, Depends(get_current_active_user)], instance
```

Comment:
why the need for the whole Annotated type with Depends - wont it execute the `get_currrent_active_user` twice?